client/src/game/draw.js

```diff
@@ -1,4 +1,8 @@
 export default function draw(context) {
   context.fillStyle = "#000000";
   context.fillRect(0, 0, context.canvas.width, context.canvas.height);
+
+  //floor
+  context.fillStyle = "#ff0167";
+  context.fillRect(0, 500, context.canvas.width, 4);
```

Comment:
You create a comment to say, that these lines relate to the floor.
In this case, it makes sense to create a function instead (functions === comments).

```suggestion
  drawFloor(context);
```

Reply:
Das review ging ja fix 😮 

You are totally right. But the draw.js is just a development js-file which will be replaced with a drawPlattformes.js and a drawPlanet.js ;) So I think I will leave it this way for now.
At the moment the black rectangle in the draw.js is the background of the canvas. This will be replaced with a background image in the canvas. And the floor-line should be invisible at the end, because there will be the planet landscape in the background :D